### ISSUE TITLE
Missing dependecies when building OSS

### ISSUE THREAD

**Issue by Mojster** [jaeksoft/opensearchserver#1829]
Hello,

In the documentation is stated that I should clone the Git repo and then install 4 custom jars (pojodbc-1.1.2.jar, icepdf-viewer-4.4.0.jar, icepdf-core-4.4.0.jar, hunspell-dren-1.0.jar) to my local Maven repo.

But in your current Git repo (master) the lib directory does not exist anymore.

**Comment by Mojster** [comment]
Tried it again today with fresh code and came further.
Build failed at gpg. I've removed it from pom and build was successful.

How do I use the generated jar/war.
Tried to move it into existing OSS downloaded from web and it throws some errors on startup.

**Comment by Mojster** [comment]
Today I did a clean git clone and installed and configured gpg. Now the build goes through. It has some errror on generating javadoc, but it's still successful.

I've replaced jar & war in current working installation on Windows and I get following error:

`
PS D:\Programs\opensearchserver> .\start.bat
Exception in thread "main" com.github.jankroken.commandline.domain.UnrecognizedSwitchException: class com.jaeksoft.searc
hlib.Server$Arguments: Doss.externalparser.classpath=D:\Programs\opensearchserver/lib/ext/*
        at com.github.jankroken.commandline.domain.OptionSet.consumeOptions(OptionSet.java:43)
        at com.github.jankroken.commandline.CommandLineParser.parse(CommandLineParser.java:39)
        at com.jaeksoft.searchlib.Server.start(Server.java:124)
        at com.jaeksoft.searchlib.Server.main(Server.java:132)
PS D:\Programs\opensearchserver>`
